### ISSUE TITLE
Live region: aria-relevant removals does not work

### ISSUE THREAD

**Issue by JAWS-test** [FreedomScientific/VFO-standards-support#289]
### Summary

Live region: aria-relevant removals does not work

1. Save as HTML file:
```
<!DOCTYPE html>
<html lang="en">
	<head>
		<meta charset="utf-8">
		<title>live region</title>
	</head>
	<body>
		<div id=1><div aria-live=polite aria-relevant=removals id=2>same element</div></div>
		<div id=3><div  id=4><div aria-live=polite aria-relevant=removals>outside</div></div></div>
		<div aria-live=polite aria-relevant=removals id=5><div  id=6>inside</div></div>

		<h2>remove node</h2>
		<button onClick="document.getElementById('1').removeChild(document.getElementById('2'));">same element</button>	
		<button onClick="document.getElementById('3').removeChild(document.getElementById('4'));">outside</button>	
		<button onClick="document.getElementById('5').removeChild(document.getElementById('6'));">inside</button>	

		<h2>hide node with hidden</h2>
		<button onClick="document.getElementById('2').setAttribute('hidden','');">same element</button>	
		<button onClick="document.getElementById('4').setAttribute('hidden','');">outside</button>	
		<button onClick="document.getElementById('6').setAttribute('hidden','');">inside</button>	

		<h2>hide node with aria-hidden=true</h2>
		<button onClick="document.getElementById('2').setAttribute('aria-hidden','true');">same element</button>	
		<button onClick="document.getElementById('4').setAttribute('aria-hidden','true');">outside</button>	
		<button onClick="document.getElementById('6').setAttribute('aria-hidden','true');">inside</button>	

		<h2>replace node</h2>
		<button onClick="document.getElementById('1').innerHTML='new1';">same element</button>	
		<button onClick="document.getElementById('3').innerHTML='new2';">outside</button>	
		<button onClick="document.getElementById('5').innerHTML='new3';">inside</button>	

	</body>
</html> 
```
2. Activate all buttons (the page must be reloaded after each heading)

### Expected result
* Consistent output between browsers
* Output only when removed inside, not with "same element" or "outside"
* Output independent of the method of removal

### Actual result
* Different output from browser to browser
* Output also with "same element" (IE 11)
* No output with "inside" (Chrome, Firefox)
* Wrong output of the removed content (Chrome, Firefox)
* Output depends on the method of removal

See also https://github.com/w3c/aria-practices/issues/78, https://github.com/FreedomScientific/VFO-standards-support/issues/279, https://github.com/w3c/aria/issues/1048

The following overview shows which output is made with which method of removal:

#### remove node
* IE: same element, inside
* Chrome: none
* Firefox: none

#### hide node (tested with hidden, display:none and aria-hidden=true)
* IE: none
* Chrome: none
* Firefox: inside

#### replace node
* IE: same element, inside
* Chrome: inside with wrong output "removed n"
* Firefox: inside with wrong output "removed n"

#### JAWS version and build number

JAWS 2019.1907.42
 
#### Operating System and version

Windows 8

#### Browser and version:

Chrome 76.0.3809.100
Firefox 68.0.2
Internet Explorer 11.0.9600.19431
